game/system/SystemThread.h

```diff
@@ -14,6 +14,7 @@
 #include <mutex>
 #include <thread>
 #include <condition_variable>
+#include "third-party/spdlog/include/spdlog/spdlog.h"
```

Comment:
can we move this to SystemThread.cpp instead?  Including headers inside of cpp files when possible can speed up compile times.